lib/csvlint/csvw_property_checker.rb

```diff
@@ -0,0 +1,754 @@
+module Csvlint
+  class CsvwPropertyChecker
+
+    def CsvwPropertyChecker.check_property(property, value, base_url, lang)
```

Comment:
You can probably just do `self.check_property` here, rather than referencing the class each time if it's a class method. Or, better still, you can wrap all your class methods like so:

```
class << self
   def check_property(property, value, base_url, lang)
... blah ...

end
```